src/vector/jitsi/index.ts

```diff
@@ -356,17 +348,21 @@ function mapLanguage(language: string): string {
 // audio input it can find, while an input of null instructs it to start muted,
 // and a non-nullish input specifies the label of a specific device to use.
 // Same for video inputs.
-function joinConference(audioInput?: string | null, videoInput?: string | null): void {
+async function joinConference(audioInput?: string | null, videoInput?: string | null): Promise<void> {
     let jwt;
     if (jitsiAuth === JITSI_OPENIDTOKEN_JWT_AUTH) {
+        // See https://github.com/matrix-org/prosody-mod-auth-matrix-user-verification
+        const openIdToken: IOpenIDCredentials = await widgetApi.requestOpenIDConnectToken();
+        logger.log("Got OpenID Connect token");
+
         if (!openIdToken?.access_token) {
```

Comment:
`openIdToken` now cannot be `null` or `undefined`, so the `?` is redundant

```suggestion
        if (!openIdToken.access_token) {
```